### PR TITLE
Reliability fixes for `fork-source-link-same-view` /2

### PR DESCRIPTION
**Pull request by fregante**
Due to https://github.com/refined-github/refined-github/pull/5494#discussion_r829019629

cc @kidonng

### CODE REVIEW

**Comment by kidonng** [comment]
Thanks for the PR! I have prepared this in a separate branch (beec305498ef1b70c9c99f76817d4e7973b8eb5f) but this is also fine.

File: source/features/fork-source-link-same-view.tsx [hunk 1]
```diff
@@ -37,7 +38,7 @@ async function getEquivalentURL(): Promise<string> {
 
 async function init(): Promise<void> {
 	// The link must always be updated/reset. This pattern ensures that the link is always updated and never fails through some conditions.
-	select<HTMLAnchorElement>(`[data-hovercard-url="/${getForkedRepo()!}/hovercard"]`)!.href = await getEquivalentURL();
+	select(`a[data-hovercard-url="/${getForkedRepo()!}/hovercard"]`)!.href = await getEquivalentURL();
```
Comment (fregante) [inline source/features/fork-source-link-same-view.tsx:41]: Huh they added support for concatenated strings? 😮 This change can probably be done to more places

Comment (kidonng) [reply source/features/fork-source-link-same-view.tsx:41]: Hmm I don't know of this is only supported recently, I just thought this looks better.

Comment (yakov116) [reply source/features/fork-source-link-same-view.tsx:41]: I think we did it where we were able to already

Comment (fregante) [reply source/features/fork-source-link-same-view.tsx:41]: There are more places. For some reason sometimes it works, others it doesn't 🤷‍♂️ 

```js
❯ rg select\<
source/options.tsx
212:                    const input = select<HTMLInputElement>('#' + feature)!;

source/features/suggest-commit-title-limit.tsx
15:     const inputField = select<HTMLInputElement>(fieldSelector)!;

source/features/highest-rated-comment.tsx
70:     const {hash} = select<HTMLAnchorElement>('a.js-timestamp', bestComment)!; // TODO: Drop generic once TypeScript sorts it out, it shouldn't be needed
```

**Pull request merged**